server/py/services/api/crud/artifacts.py

```diff
@@ -284,6 +296,45 @@ def delete_artifacts(
             db_session, name, project, tag, labels, producer_id=producer_id
         )
 
+    @classmethod
```

Comment:
why classmethod? I don't think this method is needed at all tbh just call `framework.utils.singletons.db.get_db().is_artifact_can_be_removed`